src/Controller/ActivityController.php

```diff
@@ -275,9 +275,11 @@ public function getActivityDetails(Activity $activity): Response
     {
         $user = $this->getUser();
         $rights = $this->accessRightsPolicy->canAccessActivity($activity, $user);
-        $hasAccess = $this->isGranted('ROLE_ADMIN');
+        $hasAccess = $this->isGranted('ROLE_PM');
 
-        if ($rights === false && !$hasAccess) {
+        if (($rights === false && !$hasAccess) ||
+            ($activity->getStatus() === Activity::STATUS_IN_VALIDATION &&
```

Comment:
A PM has always access to the jobs posted by users for whom they are a PM. Also here you don't check that the current user is a PM of the owner. Also you have remove admin from always having access :)